crates/cheatnet/src/cheatcodes/declare.rs

```diff
@@ -96,9 +85,15 @@ impl CheatnetState {
     }
 }
 
-fn get_class_hash(casm_contract: &str) -> Result<ClassHash> {
-    let compiled_class = serde_json::from_str::<CompiledClass>(casm_contract)?;
-    let class_hash = compiled_class.class_hash()?;
+fn get_class_hash(sierra_contract: &str) -> Result<ClassHash> {
+    let sierra_class: ContractClass = serde_json::from_str(sierra_contract)?;
+    let abi_flattened = sierra_class.abi.unwrap().json();
+    let mut sierra_contract_map: HashMap<String, Value> = serde_json::from_str(sierra_contract)?;
+    sierra_contract_map.insert("abi".to_string(), Value::String(abi_flattened));
+    let sierra_contract = serde_json::to_string_pretty(&sierra_contract_map)?;
```

Comment:
This is necessary, because scarb outputs abi as json, and blockifier expects it to just be a plain string (abi format is not enforced for cairo 1 contracts, this is the case on starknet as well).